lib/ramble/ramble/workspace/workspace.py

```diff
@@ -1990,6 +1992,253 @@ def workspace_paths(self):
 
         return self._workspace_path_replacements
 
+    def _get_scope_section(self, scope):
+        base_section = None
+        if scope == "workspace":
+            base_section = self.config_sections["workspace"]["yaml"][namespace.ramble]
```

Comment:
nit: Is there a nice way to dry out the magic string `workspace` ?

Reply:
I could, but I don't know that it really buys us much here. Really the "workspace" in config_sections means something different from the one on line 1997.

I don't think making them the same dry-ed out string is a good idea. Arguably, there are more references to `"workspace"` in here that just in these lines, so I'll make a task for someone to go in and dry them out.